src/link/styled-components.js

```diff
@@ -9,11 +9,18 @@ LICENSE file in the root directory of this source tree.
 import {styled} from '../styles/index.js';
 
 export const Link = styled('a', ({$theme}) => {
-  const {colors, typography} = $theme;
+  const {colors, typography, animation} = $theme;
   return {
     color: colors.primary400,
     ...typography.font450,
     textDecoration: 'none',
+    transition: `color ${animation.timing100} ${animation.easeOutCurve}`,
```

Comment:
Please use the longhand properties instead of a shorthand.

Reply:
fixed 👍